data/transition-sites/directgov_firekills.yml

```diff
@@ -5,9 +5,9 @@ homepage: https://www.gov.uk/government/organisations/government-digital-service
 extra_organisation_slugs:
 - department-for-communities-and-local-government
 tna_timestamp: 20121106112314
-host: firekills.direct.gov.uk
+host: www.firekills.gov.uk
```

Comment:
Switching the domains around like this won't have any effect in the Transition tool because of the way it decides which host to show as the default one for a site (it just picks the one with the lowest `id`).

We decided to add the `firekills.gov.uk` domains as aliases to the existing `firekills.direct.gov.uk` site because [the former used to redirect to the latter](https://github.com/alphagov/transition-config/pull/1099#discussion_r33062430), so they were effectively the same site originally. If that's confusing for Transition users then we could split them into two separate site configs.